project-base/docker/php-fpm/Dockerfile

```diff
@@ -60,6 +60,9 @@ ENV LC_ALL=en_US.utf8 LANG=en_US.utf8 LANGUAGE=en_US.utf8
 # overwrite the original entry-point from the PHP Docker image with our own
 COPY docker-php-entrypoint /usr/local/bin/
 
+# copy php.ini configuration
+COPY project-base/docker/php-fpm/php-ini-overrides.ini /usr/local/etc/php/php.ini
```

Comment:
this assumes that the Docker build context is the root of the monorepo, right? But the COPY above assumes the actual context is the `docker` directory. Both can't be true.